lib/src/stack/types.tsx

```diff
@@ -10,9 +13,21 @@ type Props = {
     | "xlarge"
     | "xxlarge"
     | "xxxlarge";
+  /**
+   * If true a divider is shown between children.
+   */
   divider?: boolean;
+  /**
+   * Alignment applied to children.
+   */
   align?: "start" | "center" | "end" | "baseline" | "stretch";
+  /**
+   * Component type or primitive that is rendered as the type of the root element.
```

Comment:
If you made any change based on the feedback, remember to repeat it here 🚀.

Reply:
Done!! ✌🏼